### PR TITLE
feat(css/parser): normalize declaration and important

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Normalize declartion and important, also found a bug in `swc_css_prefixer` (because we don't check case) and fixed

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_codegen/src/lib.rs [hunk 1]
```diff
@@ -2763,27 +2763,27 @@ fn minify_hex_color(value: &str) -> String {
...
                 return minified;
             }
             // 8 -> 4
             else if length == 8 && chars[6] == chars[7] {
                 let mut minified = String::with_capacity(4);
 
-                minified.push((chars[0] as char).to_ascii_lowercase());
-                minified.push((chars[2] as char).to_ascii_lowercase());
-                minified.push((chars[4] as char).to_ascii_lowercase());
-                minified.push((chars[6] as char).to_ascii_lowercase());
```
Comment (alexander-akait) [inline crates/swc_css_codegen/src/lib.rs:2779]: Forgot to remove it too, we don't need it, because we already normalize hex colors

File: crates/swc_css_codegen/src/lib.rs [hunk 2]
```diff
@@ -2763,27 +2763,27 @@ fn minify_hex_color(value: &str) -> String {
...
+                minified.push(chars[4] as char);
+                minified.push(chars[6] as char);
 
                 return minified;
             }
         }
     }
 
-    value.to_ascii_lowercase()
+    value.to_string()
```
Comment (alexander-akait) [inline crates/swc_css_codegen/src/lib.rs:2786]: Ideally rewrite it on `Cow`, but let's do it in the separate PR

**Review by kdy1 — APPROVED** [review]
swc-bump:
 - swc_css_codegen
 - swc_css_parser
 - swc_css_minifier
 - swc_css_prefixer
